<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerBuildConfigurationWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerBuildConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>111</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="specFileLabel">
     <property name="text">
      <string>RPM spec file:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::PathChooser" name="specFilePathChooser" native="true">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sfdkOptionsLabel">
     <property name="text">
      <string>Additional sfdk configuration:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::FancyLineEdit" name="sfdkOptionsLineEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Additional sfdk configuration options using the shorthand syntax '&amp;lt;name&amp;gt;[=[&amp;lt;value&amp;gt;]]' as explained by `sfdk config --help`. Subject to sh-compatible word splitting.&lt;/p&gt;&lt;p&gt;Allowed options: %1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
